mopro-ffi/src/lib.rs

```diff
@@ -226,18 +226,30 @@ pub fn arkworks_pippenger(
     .unwrap();
     Ok(benchmarks)
 }
+// #[cfg(feature = "gpu-benchmarks")]
+// pub fn trapdoortech_zprize_msm(
```

Comment:
will this be used in the future?

Reply:
not unless it's for benchmarking purpose